Send pretty HTML template when showing 403 error (when SEB denies access)

Ivanca opened the pull request:
Created backend for edxmako to allow testing, seb-403.html is heavily based on /lms/templates/static_templates/404.html

Ivanca committed:
Send pretty HTML template when showing Access-denied-must-use-seb error
Created backend for edxmako to allow testing, seb-403.html is heavily based on /lms/templates/static_templates/404.html

felipemontoya commented on seb_openedx/middleware.py:
```diff
@@ -22,11 +23,18 @@ def process_view(self, request, view_func, view_args, view_kwargs):
+
+    def render_to_response(self, template_name, dictionary=None, namespace='main', request=None, **kwargs):
+        """ Custom render_to_response implementation using configurable backend and adding template dir """
+        edxmako = get_edxmako_module()
```
This render to response method is what we really want. I suggest instead of having 
`from seb_openedx.edxapp_wrapper.get_edxmako_module import get_edxmako_module` we do 
`from seb_openedx.edxapp_wrapper.edxmako_module import render_to_response` and move all the logic there

felipemontoya reviewed (changes requested):
Overall I think this if fine. I do have some comments about where we should wrapp the edxmako module

Ivanca committed:
Fixes based on feedback
Instead of having
`from seb_openedx.edxapp_wrapper.get_edxmako_module import get_edxmako_module` we do
`from seb_openedx.edxapp_wrapper.edxmako_module import render_to_response` and moved all the logic there

felipemontoya reviewed (approved):
Nice work
